client/src/pages/MyProfile/MyProfile.js

```diff
@@ -0,0 +1,233 @@
+import React from 'react';
+import Tag from '../../components/Tag/Tag';
+import instagram from '../../assets/instagram.svg';
+import facebook from '../../assets/facebook.svg';
+import tina from '../../assets/tina.png';
+import plusSign from '../../assets/plus-sign.svg';
+import './MyProfile.css';
+import '../ProfilePage/ProfilePage.css';
+import '../CreateProfilePage/CreateProfilePage.css';
+import NavBar from '../../components/Navbar/Navbar';
+
+export default function MyProfile(props) {
```

Comment:
remember that we already have `ProfilePage` in our repo (look at `src/pages/ProfilePage` - feel free to rename it to `MyProfilePage` if you wish)

here's my suggestion:
- create a component `Profile` which takes in the existing props as well as additional props such as `editOnly` (described in my earlier comment) and `viewOnly` (for FiltersPage)
- put all the logic (states, renderEdit, renderView, etc.) in `Profile` *only*: no other file should have those functions
- as i mentioned above, refactor `CreateProfilePage` to use the newly created `Profile` component with `editOnly=true` so that we don't duplicate 233 lines of code
- refactor `ProfilePage` to use `Profile` component